### ISSUE TITLE
YouTube Skip Forward/Back Bugged

### ISSUE THREAD

**Issue by LimitedEdiition** [ReVanced/revanced-patches#3305]
### Bug description

Recently patched with all default on 18.45.41 (even uninstalled and reinstalled) and the double tap to skip forward 10 seconds and skip back 10 seconds has been showing very inconsistent behavior compared to previous YouTube versions. 

Before the newest patch, only skip back would have some issues (but those were easily solved because you could just tap once for the media controls to show up and then double tap then), but now even tapping to show the media controls has problems. It seems like it's lagging or something, despite having no problems before. I have attached a video where you can see my devices taps and you can see it is definitely not functioning smoothly or accurately at all. 

Device is Samsung Galaxy S20+ on OneUI 5.1/Android 13. 

https://imgur.com/a/jgH1IP9
https://imgur.com/a/LT68l5g

### Acknowledgements

- [X] This request is not a duplicate of an existing issue.
- [X] I have chosen an appropriate title.
- [X] All requested information has been provided properly.

**Comment by oSumAtrIX** [comment]
Follow this:

![image](https://github.com/ReVanced/revanced-patches/assets/13122796/c4706c8f-9017-4f68-a49b-ea6eeff19e1d)

**Comment by Pamilg8** [comment]
It's YouTube's problem. You can check it

**Comment by LimitedEdiition** [comment]
So I had some time to see what was going on and it seems like all of the recent YouTube apks including the recommended 18.45.41 and what Liso recommended in 18.43.45 both have the same problem. I patched the most recent working version for me (18.38.44) and it works fine (I only updated to get rid of the clicking sounds when minimizing/maximizing videos). I used all default options for all of these versions as a control. 


So it might either be YouTube's fault, or one of the patch options that isn't available for this older version which is:
Bypass URL redirects
Enable Slide to Seek
Return YouTube Dislike
Seek bar Tapping
Remove tracking query parameter. 


I won't have any more time today to narrow it down to which specific one, but I can report my findings tomorrow or maybe if someone else with the same issue might want to chime in. 


Also very interesting to see the difference in the homepage layout between 18.38.44 and the recommended. 18.45.41 only shows Shorts row once at the top, but in 38.44, it shows shorts after every few videos. Not sure if this is an intended change or not,  just a casual observation. 


Thanks.
